Modules/SubModule_Import/xASL_imp_FinishImport.m

```diff
@@ -0,0 +1,207 @@
+function x = xASL_imp_FinishImport(x)
```

Comment:
OK - Michael, can you please make this new issue? Also thinking of aggregating similar things to the same issue.

Reply:
We should add this to the big issue that moves BIDS 2 legacy to processing...

Reply:
No, in the current issue this was incorrectly moved I would say. I would move this back to the BIDS2Legacy submodule, instead of moving this to the end of the import module. The point is that this householding part is dependent on if the BIDS2Legacy was run or not, so it should stay within BIDS2Legacy.

Think of the following use cases:
* A user only uses the "import" for dicom2bids, as the user wants to take the bids asl data and run FSL instead of ExploreASL
* A user only uses the "import" for dicom2nifti, as the user wants to import DWI data and keep NIfTIs
* A user only uses the "import" for bids2legacy, as the user has asl bids already and wants to use ExploreASL

These use cases are easy to address with the previous structure, when the householding was in the submodules. Now they become more tricky, if the submodules become less "modular", as their householding is in the end of the import module.

Reply:
> No, in the current issue this was incorrectly moved I would say. I would move this back to the BIDS2Legacy submodule, instead of moving this to the end of the import module. The point is that this householding part is dependent on if the BIDS2Legacy was run or not, so it should stay within BIDS2Legacy.

You know, there is a reason why I did this. Your idea doesn't work. If you would do this, you would create a `dataPar.json` for each session, a `participants.tsv` for each session, add the GeneratedBy field multiple times, ... So please [check the code again](https://github.com/ExploreASL/ExploreASL/blob/import-%23799_Module/Modules/SubModule_Import/xASL_imp_FinishImport.m) if you still think it's a good idea.